Source/Core/DolphinQt2/Debugger/RegisterColumn.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Debugger/RegisterColumn.h"
+
+#include <QMessageBox>
+
+RegisterColumn::RegisterColumn(RegisterType type, std::function<u64()> get,
+                               std::function<void(u64)> set)
+    : m_type(type), m_get_register(get), m_set_register(set)
```

Comment:
get and set can be `std::move`d

`<utility>` needs to be included in this case.